doc/nrf/releases_and_maturity/migration/2.4.99-cs3_to_2.6.99-cs2/migration_guide_2.4.99-cs3_to_2.6.99-cs2_environment.rst

```diff
@@ -0,0 +1,331 @@
+********************
+
+Make sure you have all the required hardware and that your computer has one of the supported operating systems.
+
+Hardware
+========
+
+* nRF54H20 DK version PCA10175 v0.7.0 or v0.8.0 (ES3).
+  These are the only versions of the nRF54H20 DK compatible with |NCS| 2.6.99-cs2.
+  Check the version number on your DK's sticker to verify its compatibility with |NCS| version 2.6.99-cs2.
+* USB-C cable
+
+Software
+========
+
+On your computer, one of the following operating systems:
+
+* Microsoft Windows
+* Ubuntu Linux
+* macOS
+
+|supported OS|
+
+You also need the following:
+
+* `Git`_ or `Git for Windows`_ (on Linux and Mac, or Windows, respectively)
+* `curl`_
+
+Installing the new toolchain
+****************************
+
+You can install the new toolchain for the |NCS| v2.6.99-cs2 by running an installation script.
+Ensure that you additionally have Git and curl installed.
+
+.. tabs::
+
+   .. tab:: Windows
+
+      Follow these steps:
+
+      1. Open Git Bash.
+      #. Download and run the :file:`bootstrap-toolchain.sh` installation script file using the following command:
+
+         .. parsed-literal::
+            :class: highlight
+
+            curl --proto '=https' --tlsv1.2 -sSf https://files.nordicsemi.com/artifactory/swtools-staging/external/scripts/bootstrap-toolchain.sh | NCS_TOOLCHAIN_VERSION=v2.6.99-cs2 sh
+
+         Depending on your connection, this might take some time.
+      #. Run the following command in Git Bash:
+
+         .. parsed-literal::
+            :class: highlight
+
+            c:/ncs-lcs/nrfutil.exe toolchain-manager launch --terminal --chdir "c:/ncs-lcs/work-dir" --ncs-version v2.6.99-cs2
+
+         This opens a new terminal window with the |NCS| toolchain environment, where west and other development tools are available.
+         Alternatively, you can run the following command::
+
+            c:/ncs-lcs/nrfutil.exe toolchain-manager env --as-script
+
+         This gives all the necessary environmental variables you need to copy-paste and execute in the same terminal window to be able to run west directly there.
+
+         .. caution::
+            When working with the customer sampling release, you must always use the terminal window where the west environmental variables have been called.
+
+      If you run into errors during the installation process, delete the :file:`.west` folder inside the :file:`C:\\ncs-lcs` directory, and start over.
+
+      We recommend adding the nRF Util path to your environmental variables.
+
+
+   .. tab:: Linux
+
+      Follow these steps:
+
+      1. Open a terminal window.
+      #. Download and run the :file:`bootstrap-toolchain.sh` installation script file using the following command:
+
+         .. parsed-literal::
+            :class: highlight
+
+            curl --proto '=https' --tlsv1.2 -sSf https://files.nordicsemi.com/artifactory/swtools-staging/external/scripts/bootstrap-toolchain.sh | NCS_TOOLCHAIN_VERSION=v2.6.99-cs2 sh
```

Comment:
Update to:
`curl --proto '=https' --tlsv1.2 -sSf https://files.nordicsemi.com/artifactory/swtools/external/scripts/bootstrap-toolchain.sh | NCS_TOOLCHAIN_VERSION=v2.6.99-cs2 sh`